examples/plot_3d_and_4d_niimg.py

```diff
@@ -16,13 +16,13 @@
 print('Datasets are stored in: %r' % datasets.get_data_dirs())
 
 ###############################################################################
-# Let's now retrieve a motor contrast from a localizer experiment
-tmap_filenames = datasets.fetch_localizer_button_task()['tmaps']
-print(tmap_filenames)
+# Let's now retrieve a motor contrast from a Neurovault repository
+motor_images = datasets.fetch_neurovault_motor_task()
```

Comment:
Should I change this localizer as well or let it be neurovault ?

Reply:
It is probably a good idea to use neurovault rather than the localizer. Let us see what this looks like.